### DISCUSSION TITLE
Changing Dreambooth temp directory / [WinError 5] Access is denied

### DISCUSSION THREAD

**Discussion by ghost** [d8ahazard/sd_dreambooth_extension#1319] [Q&A]
When training with Dreambooth, I frequently get the error

`Exception training model: '[WinError 5] Access is denied: 'C:\\Users\\User\\AppData\\Local\\Temp\\tmp79aoejfy\\unet\\diffusion_pytorch_model.safetensors''.`

I think it's related to some kind of automatic saving during training. Is there any way I can just change this directory to something else, like C:\temp?

thanks

Edit: today it refused to save my checkpoint after having trained it for several hours :\

**Comment by SunixLiu** [comment]
Same issue here

**Comment by njovy** [comment]
Is there any temporary fix for this?
